src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Amazon API Gateway V2 Monitoring Integration
```

Comment:
I would change this to "Amazon API Gateway HTTP and Websockets (v2) Monitoring Integration

Reply:
@glennthomas1, i made the suggested changes.